Tools/TestWebKitAPI/Tests/WebKitCocoa/WebTransport.mm

```diff
@@ -220,10 +229,15 @@ WebTransportServer echoServer([](ConnectionGroup group) -> Task {
         "    let c = await t.createBidirectionalStream();"
         "    let w = c.writable.getWriter();"
         "    await w.write(new TextEncoder().encode('abc'));"
+        "    await w.close();"
+        "    await c.readable.getReader().cancel();"
         "    let sr = t.incomingBidirectionalStreams.getReader();"
         "    let {value: s, d} = await sr.read();"
         "    let r = s.readable.getReader();"
         "    const { value, done } = await r.read();"
+        "    await r.cancel();"
```

Comment:
To test a lot of this I think it would be useful to make it so the server can terminate connections without having been asked to by JS.